react/containers/PublishUrlInput.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { updateClaim } from '../actions/index';
+import { connect } from 'react-redux';
+import { makeGetRequest } from '../utils/xhr.js';
+import UrlMiddle from '../components/PublishUrlMiddle.jsx';
```

Comment:
You should look at some of the linting additions @IGassman has added to app, could help keep these imports consistent in path specification (among other improvements).